aws/resource_aws_emr_instance_group_test.go

```diff
@@ -406,6 +406,17 @@ func testAccAWSEmrInstanceGroupConfig(r int) string {
 	`, r, r, r, r, r, r)
 }
 
+func testAccAWSEmrInstanceGroupConfig_bid(r int) string {
```

Comment:
This is only a config that is not ran as an acceptance test.

To use this config, you should write an acceptance test, as the ones that are named `Test*` (with the capital T), generally at the top of `_test.go` files.